pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+## Description
```

Comment:
suggestion: megalinter complains about this file:
```
✅ Linted [MARKDOWN] files with [markdownlint]: Found 1 non blocking error(s) - (0.55s) (expand for details)
  - Using [markdownlint v0.33.0] https://megalinter.io/6.22.2/descriptors/markdown_markdownlint
  - MegaLinter key: [MARKDOWN_MARKDOWNLINT]
  - Rules config: [.markdownlint.json]
  - Number of files analyzed: [1]
  --Error detail:
  /github/workspace/pull_request_template.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "## Description"]
  ```
  
  Maybe it's worth fixing, or ignoring this issue?

Reply:
For now, I don't see the value in giving the title in the description, but if you want, I can add it :) @trOnk12 what do you think?

Reply:
Ok, so let me leave it as it is :)